updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: Bump JNLP Container Agent Image jenkins/inbound-agent (JDK17)
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  packerImageVersion:
+    kind: githubrelease
+    spec:
+      owner: "jenkins-infra"
+      repository: "packer-images"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"

```

Comment:
```suggestion
```

(nitpick) This source is not needed in this manifest (try to search its ID)

Reply:
@shubhamch71 What I referred as "ID" is the key `packerImageVersion` which is the unique identifier of the "updatecli source" in this YAML file.

If you search for the string `packerImageVersion` in this file (`jenkinscontroller-agents-jnlp-container-jdk17.yaml`), you won't see any other occurence: that is because the source is not used in it.

Rationale is that the targeted container image for JNLP uses the `"jenkins/inbound-agent"` Docker image which is unrelated to the Jenkins Infra Packer "all in one" image (and the reason you initially splitted the manifests).

=> As such you can remove the whole block as per my suggestion